common.js

```diff
@@ -1,10 +1,13 @@
+'use strict'
+
```

Comment:
In my change I need to add new variables. I decided to use `const`. And I cannot use `const`/`let` without `use strict`.  It was a reason why I use `use strict` here.  

Should I revert it and use only `var`?